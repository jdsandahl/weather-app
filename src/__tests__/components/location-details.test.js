import React from 'react';
import { render } from '@testing-library/react';
import LocationDetails from '../../components/location-details';

it('renders correct className applied to the element containing city and location text', () => {
    const { getByText } = render (
        <LocationDetails city="Manchester" country="UK" />
    );

    expect(getByText("Manchester, UK")).toHaveClass("location-details");
}); 