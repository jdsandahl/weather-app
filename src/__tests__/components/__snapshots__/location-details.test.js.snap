// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationDetails renders correct text against a snapshot 1`] = `
<DocumentFragment>
  <h1
    class="location-details"
  >
    Manchester, UK
  </h1>
</DocumentFragment>
`;
